packages/frontend/src/presenter/DataSources/useCrunchbaseHandler.ts

```diff
@@ -0,0 +1,40 @@
+import { useForm } from 'react-hook-form';
+import vest, { test, enforce } from 'vest';
+import { vestResolver } from '@hookform/resolvers/vest';
+
+import { useFormErrors } from '../../model/Hooks';
+import {
+  CrunchbaseForm,
+  HandlerStep,
+  UseCrunchbaseHandlerHook,
+} from '../../types';
+
+const suite = vest.create('crunchbase_handler', ({ userKey }) => {
+  test('userKey', 'User key is required', () => {
+    enforce(userKey).isNotEmpty();
+  });
+});
+
+export const useCrunchbaseHandler: UseCrunchbaseHandlerHook = ({
+  state,
+  updateState,
+  step,
+}: HandlerStep) => {
```

Comment:
Remove `HandlerStep`, you provide types for props with `UseCrunchbaseHandlerHook` type